ActindoWorkFlow/UserInterface/04a_ProcessActions.md

```diff
@@ -1,104 +1,71 @@
+    - **Error; automatic retry**   
+        Select this option to display the actions that have given an error, but will be automatically retried.
     - **Suspended**  
-        Select this option to display the actions that have been manually paused.
+        Select this option to display the actions that have been manually paused. 
     - **Process aborted**  
         Select this option to display the actions included in a process that has been manually ended.
+    - **Aborted**   
+        Select this option to display the actions that has been manually ended.
+    - **Manual**    
+        Select this option to display the actions that require a user decision.
 
-- ![Search](../../Assets/Icons/Search.png "[Search]") (Search)  
-    Click this button to display the search bar and search for an action.
-
-    > [Info] The search function can be used to search for a transition name. At least three characters must be entered.
-
-- ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh)  
-    Click this button to update the list of actions.
-
-- ![Columns](../../Assets/Icons/Columns.png "[Columns]") Columns (x)  
-    Click this button to display the columns bar and customize the displayed columns and the order of columns in the list. The *x* indicates the number of columns that are currently displayed in the list.
-
-- ![Filter](../../Assets/Icons/Filter.png "[Filter]") Filter (x)  
-    Click this button to display the filter bar and customize the active filters. The *x* indicates the number of filters that are currently active.
-
-The list displays all actions that are matching the selected status. Depending on the settings, the displayed columns may vary. All fields are read-only. Click an action to display the corresponding process in the *Process ID* view, see [Process ID](./03a_Processes.md#process-id).
-
-> [Info] When clicking an action in the *Process actions* view, the selected action is displayed in the *Actions* tab of the corresponding *Process ID* view. The *Actions* tab of the *Process ID* view is, in fact, an excerpt of the *Process actions* menu entry. Click the *Logs* tab in the *Process ID* view for detailed information about the selected process actions, see [Logs](./06a_Logs.md).
+The following functions are available for the editing toolbar. Depending on the process status, the editing toolbar displays specific buttons:
 
 - [x]    
     Select the checkbox to display the editing toolbar. If you click the checkbox in the header, all actions in the list are selected.
 
-Depending on the action status, the editing toolbar displays the following buttons:
-
 - ![Retry](../../Assets/Icons/Retry01.png "[Retry]") (Retry)  
-    Click this button to retry the action. This button is displayed if the action status is *Error*.
+    Click this button to retry the action. This button is displayed if the action status is *Error*, *Done*, or *Aborted*.
```

Comment:
Bei Done und Aborted sinnvoll? Evtl. mit Entwicklung klären, ob der Button überhaupt da hingehört in diesen Fällen?